src/SFA.DAS.AssessorService.Application.Api.External/Startup.cs

```diff
@@ -145,19 +157,19 @@ private IServiceProvider ConfigureIoC(IServiceCollection services)
 
                 if (_useSandbox)
                 {
-                    config.For<AssessorApiClientConfiguration>().Use(ApplicationConfiguration.SandboxAssessorApiAuthentication);
+                    config.For<AssessorApiClientConfiguration>().Use((AssessorApiClientConfiguration)this.Configuration.SandboxAssessorApiAuthentication);
```

Comment:
We are not using "this." syntax as it's mostly unnecessary

Reply:
Removed from latest commit and unnecessary casts.